### PR TITLE
SCRUM 63

### PR DESCRIPTION
**Pull request by sckaper**
- search bar works well
- admin can delete nfts

### CODE REVIEW

**Review by fourm1 — APPROVED** [review]
Seems good

**Comment by ouafaema** [comment]
Looks good

**Review by ouafaema — APPROVED** [review]
Looks good

File: app/admin/nfts/admin-table.tsx [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "@/components/ui/table"
+import {AdminDropDownMenu, AdminDropDownMenuProps} from "./admin-dropdown-menu"
+
+const defaultPagination: number = 0
+const maxPagination: number = 50
```
Comment (avetisk) [inline app/admin/nfts/admin-table.tsx:30]: You should centralize such values in a config file.

**Pull request merged**